src/shims/windows/handle.rs

```diff
@@ -224,6 +225,35 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
         )))
     }
 
+    fn GetStdHandle(&mut self, which: &OpTy<'tcx>) -> InterpResult<'tcx, Scalar> {
```

Comment:
This function looks to not just get a handle, it creates new handles (that must be closed) -- quit surprising given the function name. I guess the name is what the Windows API uses, but please at least add a doc comment.

Reply:
So, double-checking the docs, this is... still a somewhat wrong implementation. GetStdHandle returns a 'real' handle, but it should actually be the same one each time. You can close it, which causes _all_ handles across the whole process to become invalid. This is intentional, because in a real system you can replace the value of these handles with other ones using `SetStdHandle`.

Reply:
Yeah, that's closer to what I expected.